library/general/src/modules/Arch.rb

```diff
@@ -298,6 +293,11 @@ def board_chrp
       ppc && board_compatible == "CHRP"
     end
 
+    # true for all "CHRP" ppc boards
```

Comment:
so all chrp boards are powerNV? what is different to boarch_chrp? or just copy and paste?
Can you please also add to comment link to some documentation of this kind of ppc board?